.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
I'm sorry to say that after all the times I've tried to get the Travis-CI to work, I'm a bit done with it for the moment. I think I can take a look at it again tomorrow, with a cool head.